pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -1545,11 +1527,9 @@
       "AssemblyVersionInPackageVersion": {
         "1.0.22.0": "1.0.22",
```

Comment:
This PR is not adding additional versions, just deleting versions that were "dead".  The 1.0.21 version is not currently required by anything in the repo, should it be in the future that would generate an error and we could add it.